src/constants/appConstants.js

```diff
@@ -11,3 +11,5 @@ export const SKYPE_URL = 'skype::username?call';
 export const EMAIL_URL = 'mailto::email';
 
 export const PHONE_URL = 'tel::phone';
+
+export const NETWORK_CONNECTION_ERROR = 'Network Error';
```

Comment:
move this to `errors.js` and rename it to `NETWORK_ERROR`.

```suggestion
export const NETWORK_ERROR = 'Network Error';
```

Reply:
done